### PR TITLE
Empowered sec blob

### PR DESCRIPTION
**Pull request by rtfmwycipebcak**
pew pew pewpew pepew [sprites]

### CODE REVIEW

**Comment by Cartlord** [comment]
can we see the sprites

File: code/game/gamemodes/blob/theblob.dm [hunk 1]
```diff
@@ -349,6 +351,7 @@ var/list/blob_looks_admin = list(//Options available to admins
 	"AME" = 32,
 	"AME_new" = 64,
 	"skelleton" = 64,
+	"secblob" = 32,
```
Comment (DeityLink) [inline code/game/gamemodes/blob/theblob.dm:354]: > 32

image

**Comment by DeityLink** [comment]
kinda wish you went for 64x64 sprites there. That blob_act being a stunbaton smack is clever though. I might add a way to add custom blob_act sounds so it fits better.

**Comment by rtfmwycipebcak** [comment]
@Cartlord 
> can we see the sprites

Render004

> kinda wish you went for 64x64 sprites 

That is a lot more layers and animations and effort with little gain in this case, for what in the end is, if I'm reading the datums right,  an admin only skin.

**Comment by gurfan** [comment]
You should make it pickable when the blob starts with a sec job. Same way it's done for the other skins.

**Comment by BomberBro** [comment]
Can security or command even roll blob?

**Comment by DeityLink** [comment]
`restricted_from_jobs = list("AI", "Cyborg", "Mobile MMI", "Security Officer", "Warden", "Detective", "Head of Security", "Captain", "Head of Personnel")`

no they can't. sec offs are "enemy jobs" that Dynamic relies on to fight the blob.

**Comment by RealestEstate** [comment]
Guess we'll have to "fix" that huehuehuehue

**Pull request merged**